src/intl/en/learn-quizzes.json

```diff
@@ -324,5 +324,52 @@
+  "i001-d-label": "Work full time in blockchain infrastructure",
+  "i001-d-explanation": "Software tooling has continued to improve over time making running a node from home as a novice much more approachable. Working full-time in blockchain infrastructure is by no means a requirement to get involved.",
+  "i002-prompt": "How much ETH must be staked to operate a node?",
+  "i002-a-label": "0",
+  "i002-a-explanation": "Operating an Ethereum node does NOT require any ETH. In contrast to operating a staking validator as part of a node setup, anyone is free to run client software and sync their own sovereign copy of the blockchain—no ETH required.",
+  "i002-b-label": "8",
+  "i002-c-label": "16",
+  "i002-d-label": "32",
+  "i002-d-explanation": "Operating an Ethereum node does NOT require any ETH. In contrast to the 32 ETH required to activate a staking validator that participates directly in network consensus, anyone is free to run client software and sync their own sovereign copy of the blockchain—no ETH required.",
+  "i003-prompt": "How does running an Ethereum node benefit you as a user?",
+  "i003-a-label": "Censorship resistance",
+  "i003-a-explanation": "This is a benefit to users, but is not the only one. By running node software that communicates directly with other peers on the network, your transactions get mixed in with every other transaction your node is propagating. As such, it's nearly impossible to differentiate and censor a valid transaction that your node has shared.",
+  "i003-b-label": "Sovereignty",
+  "i003-b-explanation": "This is a benefit to users, but not the only one. By having your own copy of the Ethereum blockchain, you no longer depend on any single external party to interact with the network. You never have to ask permission to look up your balance, or to execute a transaction, and all transactions are verified using software you're running yourself. When network upgrades occur, you're in charge of whether to support the upgrade or not.",
+  "i003-c-label": "Privacy",
+  "i003-c-explanation": "This is a benefit to users, but is not the only one. Without your own node, simply looking up your account balances typically requires sending a list of your accounts from your wallet, attached to your IP address, to a third-party provider who then is being trusted to provide you with the correct information.",
+  "i003-d-label": "All of the above",
+  "i003-d-explanation": "Running a node gives you full control and sovereignty over the data you're relying on, allowing you to privately view and verify the contents of the chain, and effectively guarantee that any valid transactions are not censored.",
+  "i004-prompt": "What hard drive storage is required for an Ethereum node?",
+  "i004-a-label": "512 GB SSD",
+  "i004-a-explanation": "Currently, no client software is capable of storing the chain using only 512 GB",
+  "i004-b-label": "2 TB Rotating",
```

Comment:
Careful of combining these concepts. I'd separate type of hard disk and storage capacity

Reply:
Two 🕊️ one 🪨? It's more just a chance to use a wrong answer as a teaching point.